.vscode/settings.json

```diff
@@ -32,7 +32,7 @@
 		"editor.defaultFormatter": "esbenp.prettier-vscode"
 	},
 	"[typescriptreact]": {
-		"editor.defaultFormatter": "esbenp.prettier-vscode"
+		"editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
Is this needed due to an update from prettier?

Reply:
No, it was an accidental change by vscode. I've reverted it.